plugins/modules/ise_radius_integration_workflow_manager.py

```diff
@@ -0,0 +1,1466 @@
+              confidentiality and integrity of authentication and accounting data exchange.
+            type: str
+            choices: [KEYWRAP, RADSEC]
+          message_key:
+            description:
+            - Message key used to encrypt shared secret.
+            - Updation of message key is not possible.
+            - Required when encryption_scheme is provided.
+            - >
+              When ASCII format is selected, Message Authentication Code Key may contain
+              alphanumeric and special characters. Key must be 20 char long.
+            type: str
+          encryption_key:
+            description:
+            - Encryption key used to encrypt shared secret.
+            - Updation of encryption scheme is not possible.
+            - Required when encryption_scheme is provided.
+            - >
+              When ASCII format is selected, Encryption Key may contain
+              alphanumeric and special characters. Key must be 16 char long.
+            type: str
+          authentication_port:
+            description:
+            - Authentication port of RADIUS server.
+            - Updation of authentication port is not possible.
+            - Authentication port should be from 1 to 65535.
+            type: str
+            default: "1812"
+          accounting_port:
+            description:
+            - Accounting port of RADIUS server.
+            - Updation of accounting port is not possible.
+            - Accounting port should be from 1 to 65535.
+            type: str
+            default: "1813"
+          port:
+            description:
+            - Port of TACACS server.
+            - Updation of port is not possible.
+            - Port should be from 1 to 65535.
+            type: str
+            default: "49"
+          retries:
+            description:
+            - Number of communication retries between devices and authentication and policy server.
+            - Retries should be from 1 to 3.
+            type: str
+            default: "3"
+          timeout:
+            description:
+            - Number of seconds before timing out between devices and authentication and policy server.
+            - Timeout should be from 2 to 20.
+            type: str
+            default: "4"
+          role:
+            description:
+            - Role of authentication and policy server.
+            - Updation of role is not possible
+            type: str
+            default: secondary
+          pxgrid_enabled:
+            description:
+            - Set True to enable the Pxgrid and False to disable the Pxgrid.
+            - Pxgrid is available only for the Cisco ISE Servers.
+            - >
+              PxGrid facilitates seamless integration and information sharing across products,
+              enhancing threat detection and response capabilities within the network ecosystem.
+            type: bool
+            default: True
+          use_dnac_cert_for_pxgrid:
```

Comment:
This can be enabled only if "Enable Multiple Cisco DNA Center operation" is enabled in GUI rite?

Reply:
Can't we move this inside   cisco_ise_dtos: ?